internal/handler/msg_aggregate.go

```diff
@@ -86,7 +86,7 @@ func (h *Handler) MsgAggregate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 	db, err := h.b.Database(dbName)
 	if err != nil {
 		if backends.ErrorCodeIs(err, backends.ErrorCodeDatabaseNameIsInvalid) {
-			msg := fmt.Sprintf("Invalid namespace specified '%s.%s'", dbName, cName)
+			msg := fmt.Sprintf("Namespace %s is not a valid collection name", dbName)
```

Comment:
If this line is not covered by tests, do we need to change it?

Reply:
No! Reverted the changes.

Reply:
Hmm, I don't see the changes reverted, and the line is still marked as not covered.